lib/tail.js

```diff
@@ -23,30 +23,17 @@ function Tail(path, opts) {
       this.emit('line', str);
     });
   } else {
-    let followOpt = '-F';
-    if (process.platform === 'openbsd') {
-      followOpt = '-f';
-    }
-
-    const tail = childProcess.spawn('tail', ['-n', options.buffer, followOpt].concat(path));
-
-    tail.stderr.on('data', (data) => {
-      // If there is any important error then display it in the console. Tail will keep running.
-      // File can be truncated over network.
-      if (data.toString().indexOf('file truncated') === -1) {
-        console.error(data.toString());
-      }
+    const tail = tailStream.createReadStream(path.join(), {
+      encoding: 'utf8',
+      start: 80,
```

Comment:
I think start should be `options.buffer` here. `options.buffer` is what the user passes as a `-n` CLI param.

Reply:
:confused: we have a problem here. the `-n` parameter is defined as lines but node's createReadStream uses bytes for the start parameter. I need to look into this...

Reply:
I tested it and changing `start` to `options.buffer` worked very good.